### ISSUE TITLE
Allow setting a minimum LogEventLevel

### ISSUE THREAD

**Issue by th0mk** [CXuesong/Serilog.Sinks.Discord#2]
In the seq and console sink, you can pass a minimum `LogEventLevel`:

```
Log.Logger = new LoggerConfiguration()
                .WriteTo.Console(LogEventLevel.Error)
                .WriteTo.Seq("http://localhost:5341", LogEventLevel.Debug)
```

It would be nice if this was also possible for this Discord sink.

**Comment by CXuesong** [comment]
Released [`v0.5.0`](https://github.com/CXuesong/Serilog.Sinks.Discord/releases/tag/v0.5.0). Please post here if you come across any issue.

**Comment by th0mk** [comment]
Thank you, this works great.
